tests/sla_tests.js

```diff
@@ -206,4 +228,42 @@ describe('SLA Bridge and Channel Tester', function() {
       }, asyncDelay);
     } 
   });
+  it('should enter the application but specify an invalid SLA bridge',
+     function(done) {
+    var client = getMockClient();
+    var inbound = getMockChannel();
+    var sla = require('../lib/sla.js')(client, inbound, 'invalid').done();
+
+    incorrectBridge();
+    function incorrectBridge() {
+      setTimeout(function() {
+        if(!pastOriginate && bridgeChannels.length === 0 &&
```

Comment:
All those conditions are true after a call to afterEach so that's not actually testing what you are looking for. You have to have a way to flag that even though it was rejected, the code made it to a certain point.